src/e2e-test/java/io/cdap/plugin/bigquery/stepsdesign/BQValidationExistingTables.java

```diff
@@ -0,0 +1,119 @@
+package io.cdap.plugin.bigquery.stepsdesign;
+
+import com.esotericsoftware.minlog.Log;
+import com.google.cloud.bigquery.FieldValueList;
+import com.google.cloud.bigquery.TableResult;
+import com.google.gson.Gson;
+import com.google.gson.JsonElement;
+import com.google.gson.JsonObject;
+import io.cdap.e2e.utils.BigQueryClient;
+import io.cdap.e2e.utils.PluginPropertyUtils;
+import io.cucumber.core.logging.Logger;
+import io.cucumber.core.logging.LoggerFactory;
+
+import java.io.BufferedReader;
+import java.io.FileReader;
+import java.io.IOException;
+import java.net.URISyntaxException;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.HashMap;
+import java.util.Map;
+
+/**
+ * BigQuery Plugin Existing Table validation.
+ */
+public class BQValidationExistingTables {
+
+  private static final Logger LOG = LoggerFactory.getLogger(BQValidationExistingTables.class);
+  static Gson gson = new Gson();
+  public static boolean validateActualDataToExpectedData(String table, String fileName) throws IOException,
+    InterruptedException, URISyntaxException {
+    Map<String, JsonObject> bigQueryMap = new HashMap<>();
+    Map<String, JsonObject> fileMap = new HashMap<>();
+    Path importbqexpectedfile = Paths.get(BQValidationExistingTables.class.getResource("/" + fileName).toURI());
```

Comment:
Rename importbqexpectedfile to bqExpectedFilePath or just path

Reply:
renamed